tests/ci/ut_install.sh

```diff
@@ -10,10 +10,12 @@ go get github.com/docker/libtrust
 go get golang.org/x/lint/golint
 go get github.com/GeertJohan/fgt
 go get github.com/dghubble/sling
+set +e
```

Comment:
why do we need `set +e` at this line 13 and `set -e` at line 18? 
Is it because line 14 to line 17 all exit with 1?

Reply:
just sync the change with branch 2.7 and main.